### DISCUSSION TITLE
Maps Android SDK

### DISCUSSION THREAD

**Discussion by Psijic** [organicmaps/organicmaps#5617] [Q&A] [answered]
Hello, is it possible to use parts of organicmaps as a separate OSM map renderer in a custom commercial Android app (like Google Maps SDK)?

**Comment by biodranik** [comment, marked as answer]
Yes, if you follow the terms of the Apache 2.0 license and explicitly mention Organic Maps in your product.

Building SDKs is in our long-term plans, when we get enough resources. You may start doing it as an open-source fork that is synced with the main repo (as we are constantly improving the core). In this case, we may consider merge back your changes in the main repo somewhere in the future (if it won't break Organic Maps app, of course).

We would also appreciate any fixes/improvements contributed back into the main repo.

**Comment by Psijic** [comment]
You mean, currently the proper way is to fork the repo, subtract everything unnecessary and use it as a needed library, right? But there could be troubles with the updates from the original repo then. Looks like submodules could be a nice part of such split.
